portfolio/src/main/webapp/js/custom-script.js

```diff
@@ -53,8 +53,8 @@ function statisticsOnLoad() {
     ['Robert Fuller', 'Palmdale, California'],
     ['James Scurlock', 'Omaha, Nebraska'],
     ['Elijah McClain', 'Aurora, Colorado'],
-    ['Placeholder', 'Mountain View, CA'],
-    ['Placeholder', 'Mountain View, CA'],
+    ['Justin Howell', 'Austin, Texas'],
+    ['Jamel Floyd', 'Brooklyn, New York'],
   ];
 
   const iconicProtestData = [
```

Comment:
For all options, make sure everything is capitalized other than certain articles like a, and, the, etc. Also, only one has the date so I'd either add the date for the other options or remove it for consistency.